key-sender.js

```diff
@@ -112,7 +112,7 @@ module.exports = function() {
         return new Promise(function(resolve, reject) {
             var jarPath = path.join(__dirname, 'jar', 'key-sender.jar');
 
-            var command = 'java -jar ' + jarPath + ' ' + arrParams.join(' ') + module.getCommandLineOptions();
+            var command = 'java -jar \'' + jarPath + '\' ' + arrParams.join(' ') + module.getCommandLineOptions();
```

Comment:
I have struggled with the same problem. For me it worked when I placed double quotes instead of single quotes (using Your fix code with escaping).

Reply:
Was there a quotation on the path?

Reply:
Yes, there were single quotation marks and I had an error. I tried without any quotation, which also caused an error. Only when I put escaped double quotes (instead of single ones in Your fix) it works properly.